android/src/main/java/com/tundralabs/fluttertts/FlutterTtsPlugin.java

```diff
@@ -146,7 +146,7 @@ void setSpeechRate(float rate) {
     }
 
     Boolean isLanguageAvailable(Locale locale) {
-        return tts.isLanguageAvailable(locale) > 0;
+        return tts.isLanguageAvailable(locale) >= 0;
```

Comment:
Might be better to use the constant LANG_AVAILABLE.
`return tts.isLanguageAvailable(locale) >= TextToSpeech.LANG_AVAILABLE;`